airflow/models/baseoperatorlink.py

```diff
@@ -20,14 +20,47 @@
 from abc import ABCMeta, abstractmethod
 from typing import TYPE_CHECKING, ClassVar
 
-import attr
+import attrs
+
+from airflow.models.xcom import BaseXCom
+from airflow.utils.log.logging_mixin import LoggingMixin
 
 if TYPE_CHECKING:
     from airflow.models.baseoperator import BaseOperator
     from airflow.models.taskinstancekey import TaskInstanceKey
 
 
-@attr.s(auto_attribs=True)
+@attrs.define()
+class GenericOperatorLink(LoggingMixin):
+    """A generic operator link class that can retrieve link only using XCOMs. Used while deserializing operators."""
```

Comment:
What does “generic” mean here?

Reply:
Hmm that naming makes me wonder now.

Its more of a slim interface that the webserver APIs will call to fetch operator extra links, but the implementation is such that it will only happen through an xcom pull..

Reply:
I like `XComOperatorLink` here -> describes the purpose better. I will make this change

Reply:
Handled here https://github.com/apache/airflow/pull/46613/commits/1852c6219b6997fa7d12160a85f2f8a5d7d0b2a5